engine/runtime-parser/src/main/java/org/enso/compiler/core/ir/IrPersistance.java

```diff
@@ -370,32 +370,32 @@ protected Seq readObject(Input in) throws IOException, ClassNotFoundException {
   @ServiceProvider(service = Persistance.class)
   public static final class PersistMetadataStorage extends Persistance<MetadataStorage> {
     public PersistMetadataStorage() {
-      super(MetadataStorage.class, false, 301);
+      super(MetadataStorage.class, false, 381);
     }
 
     @Override
     @SuppressWarnings("unchecked")
     protected void writeObject(MetadataStorage obj, Output out) throws IOException {
-      var map =
-          obj.map(
-              (processingPass, data) -> {
-                var t = new Tuple2<>(processingPass.getClass().getName(), data);
-                return t;
-              });
-      out.writeInline(scala.collection.immutable.Map.class, map);
+      var map = new HashMap<String, Object>();
+      obj.map(
```

Comment:
`.map` no longer constructs `scala.collections.immutable.Map`, but just returns `java.util.List`. The `.map` operation isn't probably used anywhere than here else anyway.